packages/generator/src/generator.ts

```diff
@@ -111,12 +111,45 @@ function replaceJsxConditionals(program: Collection<j.Program>, templateValues:
   return program
 }
 
+export type Prompt = {
+  name: string
+  message: string
+  choices: any
+}
+
+export class PromptRunner {
+  protected readonly enquirer: Enquirer
+
+  constructor(protected readonly prompts: Array<Prompt>) {
+    this.enquirer = new Enquirer()
+  }
+
+  async run(): Promise<any> {
```

Comment:
Suggestioin, Instead of returning `Promise<any>` here, could we pass a generic parameter to the `PromptRunner` class, which sets the return type of this method?
e.g. `PrompRunner<T>`  and `run<T>()`

Returning `any` reduces the